npm-shrinkwrap.json

```diff
@@ -237,7 +237,591 @@
         "chokidar": {
           "version": "1.6.1",
           "from": "chokidar@>=1.6.1 <2.0.0",
-          "resolved": "https://registry.npmjs.org/chokidar/-/chokidar-1.6.1.tgz"
+          "resolved": "https://registry.npmjs.org/chokidar/-/chokidar-1.6.1.tgz",
+          "dependencies": {
+            "fsevents": {
```

Comment:
I believe this is not supposed to be here the script `shrinkwrap` should remove the `fsevents`. Please take a look at gulpfile `fixShrinkwrap`